doc/FREQUENTLY_MADE_SUGGESTIONS.md

```diff
@@ -77,6 +77,40 @@ Z-levels became a mandatory feature in 0.E experimental in order to support grou
 
 There has been an extensive optimization effort in the fall of 2020 to bring back performance, and it's been mostly successful.  Current experimentals should run faster with z-levels on than 0.E runs with z-levels off.
 
+#### We should not remove charges/bring them back: No.
+
+We are in the process of removing charges, that's a fact that won't be changed.  We should've made it long before, or never make them at the first place, but it is what it is.
+
+Main reason we do this, contrary to people's belief, is not to make the game experience harsh and boring.  The main reason we are going rid of them is because, to handle both items and charges, every instance of the code that interact with them in any way should be effectively duplicated - each version of the code works by their own rules, each interact with items in their own way, and each may (and is) maintained separately. It causes tremendous amount of difficulty for any contributor - one who tries to make something new in code, one who tries to make some new items, and one who don't even touch the items directly (like mapgen people, who just try to add item to spawn on the map)
+
+Few examples:
+If you define item without charges, it's weight and volume would be equal `weight` and `volume` field: 
+
+```
+"weight": "800 g", "volume": "200ml" = 1 item weight 800 g and has volume of 200 ml
+```
+
+But if you add charges, the volume would be divided by it's `count`:
+
+```
+"weight": "800 g", "volume": "200ml", "count": 50 = 1 item has weight 800g, and volume of 4 ml
+```
+(Same goes with price, btw)
+
+
+> Wolfram Alpha tells me that the current density of bean seeds is greater than that of our sun. I do not think beans are denser than a star. (1L beans = 77.6kg) - RenechCDDA
+
+
+It sound like an easy to catch issue, but it is not easy to resolve it if you don't know about this "feature"
+
+Next, when you use charges in item definition, you can't properly use `count` in itemgroup spawn, instead you should stick to `charges` field, that is made only and specifically for charges. trying to use `charges` to spawn item? only `count` of it would be spawned.  Trying to not use `count` to spawn charged items? nothing will spawn, because `count` is zero, therefore no item will spawn;  Do you remember that the game spawns amount of `charges` by default, if other is not specified? No one can count amount of errors that happened simply because someone forgot to specify amount of charges in itemgroup
```

Comment:
```suggestion
Next, when you use charges in item definitions, you can't properly use `count` in item group spawns, instead you should stick to the `charges` field, that is there only and specifically for charges. Trying to use `charges` to spawn item? Only `count` of it would be spawned.  Trying to not use `count` to spawn charged items? nothing will spawn, because `count` is zero, therefore no item will spawn. Do you remember that the game spawns an amount of `charges` by default, if nothing else is specified? There are many, many instances of bugs, too many to count, that occurred simply because someone forgot to specify amount of charges in item groups.
```